src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -320,14 +282,17 @@ uct_rdmacm_cm_reserved_qpn_blk_alloc(uct_rdmacm_cm_t *cm,
 }
 
 ucs_status_t
-uct_rdmacm_cm_reserved_qpn_blk_free(struct uct_rdmacm_cm_reserved_qpn_blk *blk)
+uct_rdmacm_cm_reserved_qpn_blk_destroy(struct uct_rdmacm_cm_reserved_qpn_blk *blk)
```

Comment:
use 'uct_rdmacm_cm_reserved_qpn_blk_t' instead of 'struct uct_rdmacm_cm_reserved_qpn_blk_t', in all places